pages/docs/tutorials/native/basic-kotlin-native-app.md

```diff
@@ -42,26 +42,24 @@ fun main(args: Array<String>) {
 
 ## Compiling and examining output 
 
-The next step is to compile the application, which will be done using the compiler downloaded in the first step. If we have the `bin` folder
-correctly added to the path, we should be able to invoke the compiler using
+The Kotlin compiler uses a technology known as [LLVM](https://en.wikipedia.org/wiki/LLVM) to target multiple platforms. LLVM requires as input what's know as intermediate representation or IR. This IR is
+represented by a bitcode file, which is a bitstream file format. 
```

Comment:
Not sure how the IR information is relevant here.

Reply:
Well initially this was here cause there's a bc file generated and I was aiming at explaining what this file is. Then we removed the bc reference as it's a temporary issue and will be resolved by compiler. 

Having said that, I don't think having this information is wrong...